<?xml version="1.0"?>
<launch> 

  <!--include file="$(find eurobot)/launch/params.launch"/-->
  <rosparam param="/use_sim_time">true</rosparam>
  <node pkg="rosbag" type="play" name="player" args="--clock /home/egorpristanskiy/2019-05-16-12-10-44.bag" output="screen"/>
  <group ns="visualization">
    
    <node pkg="rviz" name="$(anon rviz-1)" type="rviz"  args="-d $(find eurobot_loc)/config/visualization.rviz" output="screen"/>
    <param name="field_description" textfile="$(find eurobot_loc)/models/field.urdf" />
    <param name="field_description_semantic" textfile="$(find eurobot_loc)/models/field.urdf" />
    
  </group>

  <group ns="main_robot">
    <!-- Robot model -->
    <param name="description" command="cat $(find eurobot_loc)/models/main_robot.urdf" />
    <param name="description_semantic" command="cat $(find eurobot_loc)/models/main_robot.urdf" />
  </group>

  <group ns="secondary_robot">
    <!-- Robot model -->
    <param name="description" command="cat $(find eurobot_loc)/models/secondary_robot.urdf" />
    <param name="description_semantic" command="cat $(find eurobot_loc)/models/secondary_robot.urdf" />
  </group>

  <!--<group ns="field">-->
  <!---->
    <!--<node pkg="eurobot_sim" name="cubes_broadcaster" type="cubes_broadcaster.py"  output="screen"/>-->

  <!--</group>-->

</launch>
